tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_glusterfs_profile_info.py

```diff
@@ -476,7 +493,7 @@ def get_volume_profile_metrics(self):
         volumes = self.CLUSTER_TOPOLOGY.get('volumes', [])
         # threads = []
         for volume in volumes:
-            self.process_volume_profile_info(volume)
+            self.process_volume_profile_info(volume, self.etcd_client)
```

Comment:
why do you need to add arg `self.etcd_client`

Reply:
Ack. Will change.